test/functional/VM_Test/src/j9vm/test/threads/regression/ProcessWaitFor.java

```diff
@@ -294,8 +294,8 @@ public void testWaitForAfterInterruptInSleepByStop(){
 //	we did not break the ability to break out of waitFor.
 //
 //	public void testInterruptedInWaitfor(){
-//		Thread interruptor = new TimerThread(new Interrupter(Thread.currentThread(),null),1000);
-//		interruptor.start();
+//		Thread interrupter = new TimerThread(new Interrupter(Thread.currentThread(),null),1000);
```

Comment:
The original spelling matches other code. I think this should only be changed with the code.